cmd/runtimetest/main.go

```diff
@@ -695,7 +697,12 @@ func main() {
 		cli.StringFlag{
 			Name:  "path",
 			Value: ".",
-			Usage: "path to the configuration",
+			Usage: "Path to the configuration",
+		},
+		cli.StringFlag{
+			Name:  "platform",
```

Comment:
Should the runtimetest automatically get which platform it is running on? I think it seems there is no need to specify it manunally. What do you think?